superset-frontend/package.json

```diff
@@ -51,9 +51,9 @@
     "plugins:build-assets": "node ./scripts/copyAssets.js",
     "plugins:build-storybook": "cd packages/superset-ui-demo && npm run build-storybook",
     "plugins:chromatic": "cd packages/superset-ui-demo && npm run chromatic",
-    "plugins:create-conventional-version": "npm run prune && ./scripts/lernaVersion.sh '--conventional-commits --create-release github'",
-    "plugins:create-minor-version": "npm run prune && ./scripts/lernaVersion.sh minor",
-    "plugins:create-patch-version": "npm run prune && ./scripts/lernaVersion.sh patch",
+    "plugins:create-conventional-version": "npm run prune && lerna version --conventional-commits --create-release github --yes",
+    "plugins:create-minor-version": "npm run prune && lerna version minor --yes",
+    "plugins:create-patch-version": "npm run prune && lerna version patch --yes",
```

Comment:
did you mean to change this?

Reply:
Maybe you need to rebase?

Reply:
May I know how I can do the rebase? I noticed that package.json and package.lock.json of my files were changed(probably due to installation), thus I copied these 2 files from master branch of this repo and pasted on mine, then I did this pull request. I just compared with master branch one more time, it has line 54-56 in green. **Shall I just replace line 54-56 in green with the original line 56 in red?**

Reply:
do you have a remote tracking apache?

Reply:
if so you can go to your master branch, do a git pull apache master, then go back to this branch and do a git rebase and walk through it. It is a little confusing and can be scary so I am happy to walk you through it if you want.

Reply:
Yes, plz. Can I have your help to walk me through it? Thank you